packages/tocco-ui/src/EditableValue/typeEditors/PhoneEdit.spec.js

```diff
@@ -1,24 +1,18 @@
 import React from 'react'
-import {shallow} from 'enzyme'
+import {mount} from 'enzyme'
 
 import PhoneEdit from './PhoneEdit'
 
 describe('tocco-ui', () => {
   describe('EditableValue', () => {
     describe('typeEditors', () => {
       describe('PhoneEdit ', () => {
-        test('should display the formatted phone number in input', () => {
-          const wrapper = shallow(<PhoneEdit value="+41794733123" onChange={() => {}}/>)
-          return wrapper.instance().importLibPhoneNumber()
-            .then(() => expect(wrapper.find('input').props()['value']).to.eql('+41 79 473 31 23'))
-        })
-
         test('should call on change with phone number in e.164 format', () => {
           const onChangeSpy = sinon.spy()
-          const wrapper = shallow(<PhoneEdit value="+41794733123" onChange={onChangeSpy}/>)
+          const wrapper = mount(<PhoneEdit value="0792345678" onChange={onChangeSpy}/>)
```

Comment:
just out of curiosity: why did you change the tests data?

Reply:
Changing the telephone number helped me debugging. IMO a logical sequence of figures increases readability and looks less like a real telephone number.